modules/core/src/main/java/org/apache/ignite/internal/commandline/cache/PartitionReconciliation.java

```diff
@@ -194,23 +190,24 @@ private ReconciliationResult partitionReconciliationCheck(
                 String strVal;
 
                 switch (arg) {
-                    case FIX_MODE:
+                    case REPAIR:
                         fixMode = true;
 
-                        break;
-
-                    case FIX_ALG:
-                        strVal = argIter.nextArg(
-                            "The repair algorithm should be specified. The following " +
-                                "values can be used: " + Arrays.toString(RepairAlgorithm.values()) + '.');
-
-                        try {
-                            repairAlg = RepairAlgorithm.valueOf(strVal);
-                        }
-                        catch (IllegalArgumentException e) {
-                            throw new IllegalArgumentException(
-                                "Invalid repair algorithm: " + strVal + ". The following " +
-                                "values can be used: " + Arrays.toString(RepairAlgorithm.values()) + '.');
+                        String peekedNextArg = argIter.peekNextArg();
+
+                        if (!PartitionReconciliationCommandArg.commands().contains(peekedNextArg)) {
+                            strVal = argIter.nextArg(
+                                "The repair algorithm should be specified. The following " +
```

Comment:
Don't forget to update org.apache.ignite.internal.commandline.CommandHandlerParsingTest#testPartitionReconciliationArgumentsValidation

Reply:
fixed